mesheryctl/internal/cli/root/system/cleanup.go

```diff
@@ -24,20 +25,17 @@ var cleanupCmd = &cobra.Command{
 	Use:   "cleanup",
 	Short: "Cleanup Meshery's configuration",
 	Long:  `Reset Meshery to it's default configuration.`,
+	Args:  cobra.NoArgs,
 	Run: func(cmd *cobra.Command, args []string) {
 		resetMesheryConfig()
 	},
 }
 
-func init() {
-	rootCmd.AddCommand(cleanupCmd)
-}
-
 // resets meshery config
 func resetMesheryConfig() {
 	log.Info("Meshery resetting...")
-	if err := downloadFile(dockerComposeFile, fileURL); err != nil {
+	if err := utils.DownloadFile(utils.DockerComposeFile, fileURL); err != nil {
 		log.Fatal("Error while resetting:", err)
 	}
-	log.Info("Meshery config (" + dockerComposeFile + ") settings reset to defaults.")
+	log.Info("Meshery config (" + utils.DockerComposeFile + ") settings reset to defaults.")
```

Comment:
I've had a hard time reaching this log message in the past. I wonder if `Meshery config (" + utils.DockerComposeFile + ") reset to default settings.` might improve legibility.